tabrepo/repository/abstract_repository.py

```diff
@@ -532,3 +536,100 @@ def _convert_binary_to_multiclass(self, predictions: np.ndarray, dataset: str) -
             return np.stack([1 - predictions, predictions], axis=predictions.ndim)
         else:
             return predictions
+
+    def _convert_time_infer_s_from_sample_to_batch(self, df: pd.DataFrame):
```

Comment:
Can we move those functions to an util? Those are very specific function regarding to one analysis, if we all our utils in AbstractRepository, it will be very hard to maintain.

Reply:
Yeah, a lot of this code is currently WIP. Basically, we are prioritizing making it work first, and then once it works we will do clean up